### ISSUE TITLE
Allow definition of global styles with objects and helpers

### ISSUE THREAD

**Issue by vdanchenkov** [threepointone/glamor#48]
Currently we can do

```es6
insertRule('html, body { padding: 0 }')
```

It would be great to support same definition as in createElement helper. Something like:

```es6
global('html, body', { padding: 0 })
global('a', [ { color: 'red'}, hover({ color: 'blue' }) ])
```

**Comment by threepointone** [comment]
Good idea, I like it! Have to call it something other than `global`, but I'm for it. I'll have to get to this after a week or so, unless I see a PR :)

**Comment by threepointone** [comment]
How about this 
```jsx
insertGlobal('html, body', { padding: 0 })
insertGlobal('a', { color: 'red' })
insertGlobal('a:hover', { color: 'blue' })
```

**Comment by threepointone** [comment]
`import { insertGlobal } from 'glamor'` is available in 2.17.2, give it a spin and let me know how it feels.

**Comment by threepointone** [comment]
closing this for now. v3 will merge insertRule and insertGlobal (I think)
